### DISCUSSION TITLE
Use input for service node

### DISCUSSION THREAD

**Discussion by EvertJob** [zachowj/node-red-contrib-home-assistant-websocket#563] [Q&A]
Hi,

I would like to pass a `msg.payload` with the following content to my `call service node` data field
```json
{
	"title": "Door open",
	"message": "The front door is left open"
}
```
As far I can see this is not possible. Is there any chance this could be enabled/changed?

**Comment by zachowj** [comment]
You can your `payload` to

```
{ 
   "data": {
	"title": "Door open",
	"message": "The front door is left open"
   }
}
```

or set data field in the call-service node to `payload`

Check the documentation for valid inputs the node accepts <https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/call-service.html#input> (the target option is currently only available in the dev build)

**Reply by EvertJob** [reply]
You're great. Didn't find this on the internet, yet. Haha!
Thanks, I can build a great sub flow right now!
